master/reference/calicoctl/setup/etcdv2.md

```diff
@@ -57,18 +57,19 @@ See the table below for details on the etcdv2 specific environment variables.
 | etcdCertFile (ETCD_CERT_FILE)      | Path to the etcd client cert, e.g. `/etc/calico/cert.pem` (optional)                   | string
 | etcdCACertFile (ETCD_CA_CERT_FILE) | Path to the etcd CA file, e.g. `/etc/calico/ca.pem` (optional)                         | string
 
-> **NOTE**
->
-> 1. If you are running with TLS enabled, ensure your endpoint addresses use https
-> 2. When specifying through environment variables, the DATASTORE_TYPE environment
+> **Note**:
+> - If you are running with TLS enabled, ensure your endpoint addresses use HTTPS.
+> - When specifying through environment variables, the `DATASTORE_TYPE` environment
 >    is not required for etcdv2.
-> 3. All environment variables may also be prefixed with "CALICO_", for example
->    "CALICO_DATASTORE_TYPE" and "CALICO_ETCD_ENDPOINTS" etc. may also be used.
+> - All environment variables may also be prefixed with `CALICO_`, for example
+>    `CALICO_DATASTORE_TYPE` and `CALICO_ETCD_ENDPOINTS` etc. may also be used.
 >    This is useful if the non-prefixed names clash with existing environment
 >    variables defined on your system
-> 4. Previous versions of calicoctl supported ETCD_SCHEME and ETC_AUTHORITY environment
->    variables as a mechanism for specifying the etcd endpoints.  These variables are
->    deprecated in favor of the ETCD_ENDPOINTS list.
+> - Previous versions of `calicoctl` supported `ETCD_SCHEME` and `ETC_AUTHORITY` environment
+>    variables as a mechanism for specifying the etcd endpoints. These variables are
+>    deprecated in favor of the `ETCD_ENDPOINTS` list.
```

Comment:
Please double check if these bullets render correctly.  In particular I'm wondering if it still works with the indentation on the subsequent lines having one more space than the indentation of the first line of each bullet.

In other words, it could be that

```
> - First line of a bullet continues onto
>    more lines
```

needs to be changed to

```
> - First line of a bullet continues onto
>   more lines
```

Reply:
I have checked all the rendering in a local build. However, I will adjust this spacing for Markdown readability.